packages/fxa-admin-server/src/database/database.service.ts

```diff
@@ -114,11 +122,16 @@ export class DatabaseService implements OnModuleDestroy {
     return mergeDevicesAndSessionTokens(devices, sessionTokens, true);
   }
 
+  public async relyingParties() {
+    return this.mySqlOAuthShared.getRelyingParties();
+  }
+
   async onModuleDestroy() {
     // This tear down is important for jest tests. Without this
     // the tests will hang on completion.
     await this.connectedServicesDb.cache.close();
     await this.connectedServicesDb.db.close();
+    // await this.mySqlOAuthShared.close();
```

Comment:
Hmm, I was seeing some strange output in CI that I thought was hanging on my RP resolver tests (and they took a while locally) so I wanted to see if this helped even though I didn't think I'd need it since I didn't need knex and I'm mocking the db call [with this](https://github.com/mozilla/fxa/pull/13123/files#diff-881ae92026c80b82c32cd456d16239d109f11dc9e73d12bec1413e95d7db1576R38) (which should be a `const` that I'll also update). I didn't realize I committed this while commented out, but CI seems happy now, so I'll remove it next time I push an update.

Reply:
Interesting. ConnectedServicesDb is effectively calling close on  mySqlOAuthShared (since it's passed in there). I wonder if calling close twice becomes an issue... Anyways remove it seems good to me.

Reply:
Yeah, it was a "too long without output" error and I just figured this is what it was. I think it was something flaky or something, I've just pushed another update so I'll merge when it passes.